assets/src/web-stories-block/block/api/useUserApi.js

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import { useCallback, useMemo, useState } from 'react';
+import queryString from 'query-string';
+
+/**
+ * Internal dependencies
+ */
+import { STORIES_PER_REQUEST } from '../constants';
+
+export default function useUserApi(dataAdapter, { usersApi }) {
+  const [authorSuggestions, setAuthorSuggestions] = useState([]);
+
+  const fetchAuthors = useCallback(
+    async ({ searchTerm, page = 1, perPage = STORIES_PER_REQUEST }) => {
+      const query = {
+        context: 'edit',
+        search: searchTerm || undefined,
+        page,
+        per_page: perPage,
+      };
+
+      try {
+        const path = queryString.stringifyUrl({
+          url: usersApi,
+          query,
+        });
```

Comment:
Why not use `addQueryArgs` here?

Reply:
This code was carry forwarded from the dashboard APIs, so I did not make unnecessary changes.

Reply:
In this context, addQueryArgs is already on the page, so doesn't hurt to use it.

Reply:
not quite sure, how is `addQueryArgs` is already present here?

Reply:
I think what he means is that we can probably safely use `addQueryArgs` from `@wordpress/url` as we already use it in the old Gutenberg block and is loaded by Gutenberg already.

Although I do understand your point regarding unnecessary changes, it seems like in this instance with such a small file it should be OK.

Reply:
aha... makes sense, i'll update this to use the `addQueryArgs` no need to load extraneous dependency in this case then.

Reply:
Made chages.